src/style.css

```diff
@@ -1,3 +1,133 @@
+:root {
+  --link-color: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font-family for better cross-browser compatibility. For example, 'Roboto', Arial, sans-serif.